test/AxelarGatewayMultisig.js

```diff
@@ -893,5 +893,59 @@ describe('AxelarGatewayMultisig', () => {
           });
       });
     });
+
+    describe('send token from gateway', () => {
+      it('should burn token and emit an event', async () => {
```

Comment:
```suggestion
    describe('sendToken, () => {
      it('should burn internal token and emit an event', async () => {
```
so that you test all kinds of things about sendToken in the describe

Reply:
need to also test external token here